mongodb/Module.md

```diff
@@ -1,37 +1,27 @@
-# Ballerina MongoDB Module
-
-Connects to MongoDB from ballerina 
-
-## Module Overview
-
+## Overview
 The Mongo DB connector allows you to connect to a Mongo DB from Ballerina and perform various operations such as `getDatabaseNames`, `getCollectionNames`, `count`, `listIndices`, `find`, `insert`, `update`, and `delete`.
 
-## Prerequisites
+This module supports Ballerina Swan Lake Beta2 version.
+
+## Configuring connector
+### Prerequisites
 
 * A mongodb with username and password
 
 * Java 11 Installed <br/> Java Development Kit (JDK) with version 11 is required.
```

Comment:
Download and install Java 11...
Please use markdown syntax for bullet points.

Reply:
Fixed by 89976637b8fed8c9a16842370da109f71b035569.